app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -391,7 +412,10 @@ def process_enrollment_response(enrollment, response)
 
     case response['status']
     when IPP_STATUS_PASSED
-      if SUPPORTED_ID_TYPES.include?(response['primaryIdType'])
+      if enrollment.capture_secondary_id_enabled &&
+         !SUPPORTED_ID_TYPES.include?(response['secondaryIdType'])
```

Comment:
We still need to check if `response['primaryIdType']` is supported when `capture_secondary_id_enabled` is enabled. i.e. in addition to the `response['secondaryIdType']` check.

Reply:
i think the `elsif` on line 417 should capture that, no?